package.json

```diff
@@ -1,6 +1,9 @@
 {
-  "author": "You <you@example.com>",
-  "name": "rust-webpack-template",
+  "name": "extract-my-file",
```

Comment:
beats rust-webpack-template for sure. Yet I'd like to have a name which would include shrinking just as much as expanding

Reply:
That's true. Just wanted it aligned with the current website title first. ;-)